system_suitesparse/package.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<package>
+  <name>system_suitesparse</name>
+  <version>0.0.1</version> 
+  <description>CMake scripts finding system suitesparse library</description>
+  <maintainer email="developers@sevensense.ch">Sevensense developers</maintainer>
+  <license>closed</license>
```

Comment:
Please make this like the other package in the same repo and ASL maintainer (maybe my mavt address?) since it is hosted as part of ASL's github that is probably the right choice.